src/services/vehicles.ts

```diff
@@ -0,0 +1,43 @@
+import { Vehicle } from '../models';
+import { VehicleInfo } from '../models/vehicle-info';
+import { FuelInfo } from '../models/fuel-info';
+
+export class Vehicles {
+    public static async insert(vehicle: VehicleInfo, fuelInfo: FuelInfo): Promise<number> {
```

Comment:
This entire method will not work for Norwegian licences

Reply:
Norwegian sightings are also not stored in the database. I'd say that is a separate issue from this PR